app/models/user.js

```diff
@@ -22,28 +22,21 @@ module.exports = (sequelize, DataTypes) => {
         allowNull: false,
         unique: true,
         validate: {
-          is: '^[a-zA-Z]+.[a-zA-Z]+@wolox.+[(co|ar|cl|mx)]$'
+          is: '^[a-zA-Z]+.[a-zA-Z]+@wolox.+((co)|(ar)|(mx))$'
         }
       },
       password: {
         type: DataTypes.STRING,
         allowNull: false,
         validate: {
-          isAlphanumeric: true,
-          len: [8, 30]
+          len: [8, 61]
         }
-      },
```

Comment:
Why remove the timestamps?

Reply:
Because i redefined the model and put them in the migration instead, then, i know that the db will handle those timestamps.